fe/fe-core/src/main/java/com/starrocks/common/proc/FrontendsProcNode.java

```diff
@@ -147,7 +147,11 @@ private static boolean isJoin(List<InetSocketAddress> allFeHosts, Frontend fe) {
             if (InetAddressValidator.getInstance().isValidInet4Address(fe.getHost())) {
                 realHost = addr.getAddress().getHostAddress();
             } else {
-                realHost = addr.getAddress().getHostName();
+                try {
+                    realHost = addr.getAddress().getHostName();
+                } catch (Exception e) {
+                    LOG.warn("Fail to get address hostname of fe: " + addr.toString() + " msg:" + e.toString());
```

Comment:
```suggestion
                    LOG.warn("Fail to get address hostname of fe: {} msg: {}", addr, e);
```
the last e is used to print the error stack.

Reply:
done